pom.xml

```diff
@@ -649,11 +650,6 @@
         <artifactId>error_prone_annotations</artifactId>
         <version>${errorprone.version}</version>
       </dependency>
-      <dependency>
-        <groupId>com.google.protobuf</groupId>
```

Comment:
This is covered by `protobuf-bom`.

Reply:
This was added in https://github.com/spotify/styx/pull/1040. Maybe unintentionally? @narape